src/layouts/LinkReport/components/LinkReportModal/LinkReportModal.tsx

```diff
@@ -0,0 +1,549 @@
+                      1,
+                      Math.ceil(detailedErrorArr.length / 5)
+                    )}
+                    onPageChange={(newPage: number) => setPageNum(newPage)}
+                  />
+                </Box>
+              </HStack>
+              <TableContainer
+                mb="1.5rem"
+                width="100%"
+                backgroundColor="utility.ui"
+                borderRadius="8px"
+                border="1px"
+                borderColor="base.divider.medium"
+              >
+                <Table variant="simple">
+                  <Thead>
+                    <Tr>
+                      <Th textAlign="left" width="9rem" padding="0">
+                        <HStack
+                          height="3.5rem"
+                          padding=".375rem 1rem"
+                          gap=".5rem"
+                          fontFamily={typography.fontFamilies.inter}
+                        >
+                          <Text textStyle="subhead-2" textTransform="none">
+                            Type
+                          </Text>
+                        </HStack>
+                      </Th>
+                      <Th textAlign="left" padding=".375rem 1rem">
+                        <Text
+                          textStyle="subhead-2"
+                          fontFamily={typography.fontFamilies.inter}
+                          textTransform="none"
+                        >
+                          Reference and Error
+                        </Text>
+                      </Th>
+                    </Tr>
+                  </Thead>
+                  {!isBrokenLinksLoading && (
+                    <Tbody>
+                      {detailedErrorArr
+                        .slice((pageNum - 1) * 5, pageNum * 5)
+                        .map((link) => {
+                          const isBrokenLink = link.type === "broken-link"
+                          return (
+                            <Tr
+                              key={link.linkToAsset}
+                              borderTop="1px"
+                              borderColor="base.divider.medium"
+                            >
+                              <Td width="9rem" border="0" padding="0px 1rem">
+                                <Flex
+                                  height="100%"
+                                  align="center"
+                                  justify="left"
+                                >
+                                  <Text
+                                    textStyle="subhead-2"
+                                    fontFamily={typography.fontFamilies.inter}
+                                  >
+                                    {getErrorText(link)}
+                                  </Text>
+                                </Flex>
+                              </Td>
+                              <Td align="left" border="0px" padding="0px">
```

Comment:
check if defaults are not 0

Reply:
Seems like the default values are not 0, hence the padding="0px"